<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: Akrobat;
	font-size: 26px;
}

QFrame{
background:white;
border-radius:15px;
}

QLabel{
color:#29465B;
font-size:40px;
background:transparent;
}

QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1131</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="commandes">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>490</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#3794AB;
font-size:30px;
border-radius:45px;
color:white;
}

QPushButton:hover{
color:#3794AB;
border-radius:45px;
background:white;
font-size:30px;
}</string>
      </property>
      <property name="text">
       <string>Commandes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="produits">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#3794AB;
font-size:30px;
border-radius:45px;
color:white;
}

QPushButton:hover{
color:#3794AB;
border-radius:45px;
background:white;
font-size:30px;
}</string>
      </property>
      <property name="text">
       <string>Produits</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reparations">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>280</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#3794AB;
font-size:30px;
border-radius:45px;
color:white;
}

QPushButton:hover{
color:#3794AB;
border-radius:45px;
background:white;
font-size:30px;
}</string>
      </property>
      <property name="text">
       <string>Reparations</string>
      </property>
     </widget>
     <widget class="QPushButton" name="employes">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>280</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#3794AB;
font-size:30px;
border-radius:45px;
color:white;
}

QPushButton:hover{
color:#3794AB;
border-radius:45px;
background:white;
font-size:30px;
}</string>
      </property>
      <property name="text">
       <string>Employ√©s</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clients">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>490</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#3794AB;
font-size:30px;
border-radius:45px;
color:white;
}

QPushButton:hover{
color:#3794AB;
border-radius:45px;
background:white;
font-size:30px;
}</string>
      </property>
      <property name="text">
       <string>Clients</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1091</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#ECA966;
font-size:70px;
border-radius:60px;
color:white;
}</string>
      </property>
      <property name="text">
       <string>Savvys</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>-20</y>
        <width>1181</width>
        <height>761</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#BDD3DA;
font-size:70px;
border-radius:60px;
color:white;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>pushButton_5</zorder>
     <zorder>commandes</zorder>
     <zorder>produits</zorder>
     <zorder>reparations</zorder>
     <zorder>employes</zorder>
     <zorder>clients</zorder>
     <zorder>pushButton</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1091</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#ECA966;
font-size:70px;
border-radius:60px;
color:white;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>391</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:70px;
color:white;
}</string>
      </property>
      <property name="text">
       <string>Commandes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>1021</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:black;
font-size:70px;
border-radius:60px;
color:white;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>1021</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:white;
font-size:70px;
border-radius:60px;
color:white;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>-20</y>
        <width>1181</width>
        <height>761</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#BDD3DA;
font-size:70px;
border-radius:60px;
color:white;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="affichecommandes">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>701</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>250</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>n7eb nfasa5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>350</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>modifi</string>
      </property>
     </widget>
     <zorder>pushButton_6</zorder>
     <zorder>pushButton_7</zorder>
     <zorder>label</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>affichecommandes</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_8</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="background">
      <property name="geometry">
       <rect>
        <x>-50</x>
        <y>0</y>
        <width>1181</width>
        <height>761</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#BDD3DA;
font-size:70px;
border-radius:60px;
color:white;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>1091</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#ECA966;
font-size:70px;
border-radius:60px;
color:white;
}</string>
      </property>
      <property name="text">
       <string>Ajouter une commanes</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateCom">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>380</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>380</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date sortie </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>450</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouteCom">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>580</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Anuller">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>580</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Anuller</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="localisation">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>270</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_co">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>460</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>bonk</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLineEdit" name="idsup">
      <property name="geometry">
       <rect>
        <x>282</x>
        <y>279</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>het id bech nfasar </string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletebutton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>430</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>fasa5</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5"/>
    <widget class="QWidget" name="page_6">
     <widget class="QLineEdit" name="editcom">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>270</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="modifercom">
      <property name="geometry">
       <rect>
        <x>464</x>
        <y>482</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>badal</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QLineEdit" name="idcom">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>160</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="localisationM">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>260</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="date">
      <property name="geometry">
       <rect>
        <x>412</x>
        <y>369</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="idclientm">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>550</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="etatm">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>460</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>650</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>say√©</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>39</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
